tasks/files/snapshot.py

```diff
@@ -185,14 +204,23 @@ def get_snapshot_name(lv_name, prefix, suffix):
 
 
 def lvm_lv_exists(vg_name, lv_name):
-    lvs_command = ["lvs", "--reportformat", "json", vg_name + "/" + lv_name]
+    vg_exists = False
+    lv_exists = False
+    # check for VG
+    lvs_command = ["lvs", "--reportformat", "json", vg_name]
```

Comment:
is the `--reportformat json` used for debugging in `run_command`?

Reply:
It requests LVM to output in JSON.

Reply:
I see.  I don't use it in run_command in this case.  If the volume exists and the command works the CLI return 0 (other non-zer).  Would you prefer I remove the "--reportformat" parameter?

Reply:
> I see. I don't use it in run_command in this case. If the volume exists and the command works the CLI returns 0 (otherwise non-zero). Would you prefer I remove the "--reportformat" parameter?

Yes, please